templates/pgweb/meta.yaml

```diff
@@ -0,0 +1,32 @@
+name: pgweb
+description: Cross-platform client for PostgreSQL databases
+instructions: null
+changeLog:
+  - date: 2023-3-15
+    description: first release
+links:
```

Comment:
You could also add the Demo page `https://pgweb-demo.fly.dev`.

Reply:
not implemented